ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-chain.yaml

```diff
@@ -0,0 +1,16 @@
+chain:
+  as: ipi-conf-nutanix
+  steps:
+  - ref: ipi-conf-nutanix-context
+  - ref: ipi-conf-nutanix-dns
+  - ref: ipi-conf
+  - ref: ipi-conf-nutanix
+  - ref: ipi-conf-nutanix-manual-creds
```

Comment:
Some StatefulSet tests in `openshift-e2e-test` require storage + default storage class to be available. I recommend to install also your CSI driver as a new step openshift-e2e-test, say `ipi-conf-nutanix-csi`. Just put all manifests that OLM + your CSI driver needs into `$SHARED_DIR/` with names matching `manifest_*.yaml` and CI will add them to installation manifests and installer will create them. During installation, OLM should find them and install the operator.

Alternatively, you can create a new step *after* installation but before running any e2e tests in `openshift-e2e-nutanix-workflow.yaml`. You have a live cluster available at that time and you can do anything you want to install + configure the CSI driver.

Reply:
Thank you, @jsafrane!
We're working on CSI install part and planned to make it in separate PR.

Reply:
CSI step added.